### PR TITLE
Break up the serverless loader into typed handlers

### LINKED ISSUE #19071: Strongly type Next.js' serverless loader
It's currently incorrectly using `sendPayload` (sending a boolean instead of object) and missing behavior. We should fix this via TypeScript.

### PR DESCRIPTION
**Pull request by ijjk**
This breaks up the `serverless-loader` into typed handlers to allow for easier maintenance and better type-checking/linting. 

Closes: https://github.com/vercel/next.js/issues/19071

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| buildDuration | 10s | 9.9s | -164ms |
| nodeModulesSize | 84.9 MB | 85 MB | ⚠️ +18.4 kB |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.228 | 2.278 | ⚠️ +0.05 |
| / avg req/sec | 1122.09 | 1097.23 | ⚠️ -24.86 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.303 | 1.333 | ⚠️ +0.03 |
| /error-in-render avg req/sec | 1918.1 | 1875.81 | ⚠️ -42.29 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| 677f882d2ed8..8b81.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-fbd7082..8b7a.js gzip | 6.54 kB | 6.54 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59 kB | 59 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| _app-3b0cf13..85f8.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-6f635..c393.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4ffc3..9e0f.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-b618194..5477.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 8.01 kB | 8.01 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| Overall change | 321 B | 321 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| index.html gzip | 614 B | 614 B | ✓ |
| link.html gzip | 621 B | 621 B | ✓ |
| withRouter.html gzip | 608 B | 608 B | ✓ |
| Overall change | 1.84 kB | 1.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| buildDuration | 11.6s | 11.8s | ⚠️ +128ms |
| nodeModulesSize | 84.9 MB | 85 MB | ⚠️ +18.4 kB |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| 677f882d2ed8..8b81.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-fbd7082..8b7a.js gzip | 6.54 kB | 6.54 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59 kB | 59 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| _app-3b0cf13..85f8.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-6f635..c393.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4ffc3..9e0f.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-b618194..5477.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 8.01 kB | 8.01 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| Overall change | 321 B | 321 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| _error.js | 915 kB | 996 kB | ⚠️ +81.2 kB |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 915 kB | 996 kB | ⚠️ +81.3 kB |
| link.js | 973 kB | 1.05 MB | ⚠️ +77.2 kB |
| routerDirect.js | 966 kB | 1.04 MB | ⚠️ +77.2 kB |
| withRouter.js | 966 kB | 1.04 MB | ⚠️ +77.2 kB |
| Overall change | 4.74 MB | 5.13 MB | ⚠️ +394 kB |

</details>

</details>
Commit: 06ddf41000c0b3e5197d444815735dd1cd8db6dd

**Comment by ijjk** [comment]
## Failing test suites 
Commit: 06ddf41000c0b3e5197d444815735dd1cd8db6dd

**test/integration/serverless-runtime-configs/test/index.test.js** 
- Serverless runtime configs > experimental-serverless-trace mode > should support runtime configs in serverless mode (production)
- Serverless runtime configs > legacy serverless mode > should support runtime configs in serverless mode (production)

<details>
<summary>Expand output</summary>

  ● Serverless runtime configs › legacy serverless mode › should support runtime configs in serverless mode (production)

    expect(received).toEqual(expected) // deep equality

    - Expected  - 3
    + Received  + 1

      Object {
    -   "publicRuntimeConfig": Object {
    -     "another": "thing",
    -   },
    +   "publicRuntimeConfig": Object {},
        "serverRuntimeConfig": Object {},
      }

      110 | 
      111 |     expect(JSON.parse(ssrConfig)).toEqual(expectedSsrConfig)
    > 112 |     expect(JSON.parse(clientConfig)).toEqual(expectedClientConfig)
          |                                      ^
      113 | 
      114 |     const appSsrConfig = ssr$('#app-config').text()
      115 |     const appClientConfig = client$('#app-config').text()

      at testRuntimeConfig (integration/serverless-runtime-configs/test/index.test.js:112:38)
      at Object.<anonymous> (integration/serverless-runtime-configs/test/index.test.js:147:5)

  ● Serverless runtime configs › experimental-serverless-trace mode › should support runtime configs in serverless mode (production)

    expect(received).toEqual(expected) // deep equality

    - Expected  - 3
    + Received  + 1

      Object {
    -   "publicRuntimeConfig": Object {
    -     "another": "thing",
    -   },
    +   "publicRuntimeConfig": Object {},
        "serverRuntimeConfig": Object {},
      }

      110 | 
      111 |     expect(JSON.parse(ssrConfig)).toEqual(expectedSsrConfig)
    > 112 |     expect(JSON.parse(clientConfig)).toEqual(expectedClientConfig)
          |                                      ^
      113 | 
      114 |     const appSsrConfig = ssr$('#app-config').text()
      115 |     const appClientConfig = client$('#app-config').text()

      at testRuntimeConfig (integration/serverless-runtime-configs/test/index.test.js:112:38)
      at Object.<anonymous> (integration/serverless-runtime-configs/test/index.test.js:147:5)

</details>

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| buildDuration | 10.5s | 10.2s | -271ms |
| nodeModulesSize | 84.9 MB | 85 MB | ⚠️ +18.7 kB |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.279 | 2.288 | ⚠️ +0.01 |
| / avg req/sec | 1097.05 | 1092.8 | ⚠️ -4.25 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.384 | 1.345 | -0.04 |
| /error-in-render avg req/sec | 1805.77 | 1858.78 | +53.01 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| 677f882d2ed8..8b81.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-fbd7082..8b7a.js gzip | 6.54 kB | 6.54 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59 kB | 59 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| _app-3b0cf13..85f8.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-6f635..c393.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4ffc3..9e0f.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-b618194..5477.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 8.01 kB | 8.01 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| Overall change | 321 B | 321 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| index.html gzip | 614 B | 614 B | ✓ |
| link.html gzip | 621 B | 621 B | ✓ |
| withRouter.html gzip | 608 B | 608 B | ✓ |
| Overall change | 1.84 kB | 1.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| buildDuration | 12.2s | 12.1s | -134ms |
| nodeModulesSize | 84.9 MB | 85 MB | ⚠️ +18.7 kB |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| 677f882d2ed8..8b81.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-fbd7082..8b7a.js gzip | 6.54 kB | 6.54 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59 kB | 59 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| _app-3b0cf13..85f8.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-6f635..c393.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4ffc3..9e0f.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-b618194..5477.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 8.01 kB | 8.01 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| Overall change | 321 B | 321 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| _error.js | 915 kB | 996 kB | ⚠️ +81.3 kB |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 915 kB | 996 kB | ⚠️ +81.4 kB |
| link.js | 973 kB | 1.05 MB | ⚠️ +77.3 kB |
| routerDirect.js | 966 kB | 1.04 MB | ⚠️ +77.3 kB |
| withRouter.js | 966 kB | 1.04 MB | ⚠️ +77.3 kB |
| Overall change | 4.74 MB | 5.13 MB | ⚠️ +395 kB |

</details>

</details>
Commit: f1e0049a2c822110f1588024bd9462f4a6a1a6f1

File: packages/next/next-server/lib/router/utils/route-regex.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-interface Group {
+export interface Group {
```
Comment (lfades) [inline packages/next/next-server/lib/router/utils/route-regex.ts:1]: Is this type being used outside this file?

**Review by lfades — APPROVED** [review]
LGTM!

File: packages/next/next-server/lib/router/utils/route-regex.ts (hunk 1 above)
Comment (ijjk) [reply packages/next/next-server/lib/router/utils/route-regex.ts:1]: Not directly, it seems TypeScript needs it to be exported if used implicitly through `ReturnType<>`

Comment (lfades) [reply packages/next/next-server/lib/router/utils/route-regex.ts:1]: Makes sense, that may be a TS requirement when for generating `.d.ts` files.

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| buildDuration | 8.7s | 8.7s | -33ms |
| nodeModulesSize | 84.9 MB | 85 MB | ⚠️ +18.7 kB |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.084 | 2.065 | -0.02 |
| / avg req/sec | 1199.41 | 1210.75 | +11.34 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.359 | 1.302 | -0.06 |
| /error-in-render avg req/sec | 1838.93 | 1919.63 | +80.7 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| 677f882d2ed8..8b81.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-fbd7082..8b7a.js gzip | 6.54 kB | 6.54 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59 kB | 59 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| _app-3b0cf13..85f8.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-6f635..c393.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4ffc3..9e0f.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-b618194..5477.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 8.01 kB | 8.01 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| Overall change | 321 B | 321 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| index.html gzip | 614 B | 614 B | ✓ |
| link.html gzip | 621 B | 621 B | ✓ |
| withRouter.html gzip | 608 B | 608 B | ✓ |
| Overall change | 1.84 kB | 1.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| buildDuration | 10.3s | 10.4s | ⚠️ +50ms |
| nodeModulesSize | 84.9 MB | 85 MB | ⚠️ +18.7 kB |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| 677f882d2ed8..8b81.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-fbd7082..8b7a.js gzip | 6.54 kB | 6.54 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59 kB | 59 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| _app-3b0cf13..85f8.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-6f635..c393.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4ffc3..9e0f.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-b618194..5477.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 8.01 kB | 8.01 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| Overall change | 321 B | 321 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js serverless-loader-types | Change |
| - | - | - | - |
| _error.js | 915 kB | 996 kB | ⚠️ +81.3 kB |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 915 kB | 996 kB | ⚠️ +81.4 kB |
| link.js | 973 kB | 1.05 MB | ⚠️ +77.3 kB |
| routerDirect.js | 966 kB | 1.04 MB | ⚠️ +77.3 kB |
| withRouter.js | 966 kB | 1.04 MB | ⚠️ +77.3 kB |
| Overall change | 4.74 MB | 5.13 MB | ⚠️ +395 kB |

</details>

</details>
Commit: 1388066fd21f17b61cf634288415f2f7012fe3e2

**Review by timneutkens — APPROVED** [review]

**Pull request merged**